src/fastoad/io/configuration/tests/test_configuration.py

```diff
@@ -336,3 +341,108 @@ def test_set_optimization_definition(cleanup):
         conf_dict_opt = conf_dict["optimization"]
         # Should be equal
         assert optimization_conf == conf_dict_opt
+
+
+def test_sys_paths():
+    # Create a temporary configuration file
+    config_data = {
+        "input_file": "./inputs.xml",
+        "output_file": "./outputs.xml",
+        "model": {},
+        "sys_paths": ["/path/to/local/code1", "/path/to/local/code2"],
+    }
+
+    with tempfile.NamedTemporaryFile(delete=False, mode="w", suffix=".yaml") as temp_config_file:
+        yaml.dump(config_data, temp_config_file)
+        temp_config_file_path = temp_config_file.name
+
+    # Instantiate the configurator and load the configuration
+    configurator = FASTOADProblemConfigurator()
+    configurator.load(temp_config_file_path)
+
+    # Check if the paths are added to sys.path
+    for path in config_data["sys_paths"]:
+        assert path in sys.path
+
+    # Cleanup
+    sys.path = [p for p in sys.path if p not in config_data["sys_paths"]]
+
+    # Remove the temporary file
+    os.remove(temp_config_file_path)
+
+
+def test_imports_handling():
+    """
+    Tests the handling of imports in the configuration file
+    """
+    conf = FASTOADProblemConfigurator()
+    conf.load(DATA_FOLDER_PATH / "conf_with_imports.yml")
+
+    # Check that the classes are correctly imported and stored
+    assert "MyDriver1" in conf._imported_classes
+    assert "MyDriver2" in conf._imported_classes
+
+    # Check that _om_eval can use the imported classes
+    result1 = conf._om_eval("MyDriver1()")
+    result2 = conf._om_eval("MyDriver2()")
+
+    # Imports are done here to verify that the classes are correctly imported
+    from .data.to_be_imported.my_driver_1 import MyDriver1
+    from .data.to_be_imported.my_driver_2 import MyDriver2
+
+    assert conf._imported_classes["MyDriver1"] == MyDriver1
+    assert conf._imported_classes["MyDriver2"] == MyDriver2
+
+    # Check that _om_eval can use the imported classes
+    assert isinstance(result1, MyDriver1)
+    assert isinstance(result2, MyDriver2)
+
+
+def test_driver_configuration():
+    # Test new syntax
+    config_data_new = {
+        "input_file": "./inputs.xml",
+        "output_file": "./outputs.xml",
+        "model": {},
+        "driver": {
+            "instance": "om.ScipyOptimizeDriver(optimizer='COBYLA')",
+            "options": {"maxiter": 100, "tol": 1e-2},
+        },
```

Comment:
You miss a custom driver attribute here, like `advanced_options`.

Reply:
Corrected [415aeec](https://github.com/fast-aircraft-design/FAST-OAD/pull/559/commits/415aeeca25f2e07ead12e0195b36b10a5992f7d3)